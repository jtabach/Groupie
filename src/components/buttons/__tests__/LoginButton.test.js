import React from 'react';
import renderer from 'react-test-renderer';
import { StaticRouter } from 'react-router';
import LoginButton from '../LoginButton';

test('LoginButton Link matches snapshot', () => {
  const component = renderer.create(
    <StaticRouter location='/' context={{}}>
      <LoginButton />
    </StaticRouter>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
