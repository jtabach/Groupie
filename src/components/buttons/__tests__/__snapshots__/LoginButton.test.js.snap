// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginButton Link matches snapshot 1`] = `
<a
  href="/login"
  onClick={[Function]}
>
  Login
</a>
`;
